### ISSUE TITLE
Clarify docs on simple use

### ISSUE THREAD

**Issue by tschaub** [mantoni/consolify#6]
This looks like a great project, but I'm puzzled by the output when trying to generate a simple page.

```
$ npm install browserify consolify
$ echo "console.log('hello')" > hello.js
$ browserify --plugin consolify hello.js > hello.html
$ cat hello.html 
(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
console.log('hello')

},{}]},{},[1]);
```

That's the same output I get when bundling without the consolify plugin.  It looks like `listener.then` is not getting called.  Before digging any further, I thought I'd check if I was missing something obvious.

**Comment by tschaub** [comment]
I now see 016688279cd5a8e41008c6af412bd13bcb88f82e.  Will look at the [1.0 readme](https://github.com/mantoni/consolify/blob/v1.0.0/README.md) to see if I can work with the published version.

**Comment by mantoni** [comment]
Oh, sorry for the confusion. I should have pushed those recent commits to a branch.

**Comment by tschaub** [comment]
No problem.  I'll keep an eye out for a future release.  Looks like a great project.

**Comment by mantoni** [comment]
After a bit more testing I released it as `v2.0.0`. Please have another look.

You might also be interested in the latest [Mochify v2.8.0](https://www.npmjs.com/package/mochify) which uses this module.
